.github/workflows/build.yml

```diff
@@ -42,5 +43,5 @@ jobs:
         working-directory: packages/${{ matrix.package }}
 
       - name: Upload coverage to codecov
-        run: curl -s https://codecov.io/bash | bash
+        run: curl -s https://codecov.io/bash | bash -s -- -s coverage
```

Comment:
The CI probably should start usinng `melos` like with Riverpod, to not have to have a `path:` dependency

Reply:
this is working, I would let's migrate to Melos in another PR please to avoir delaying this, I already don't know how to rebase master with the current state as everything moved...